modules/prometheus/src/main/scala/com/evolutiongaming/skafka/producer/PrometheusProducerMetrics.scala

```diff
@@ -54,7 +55,7 @@ object PrometheusProducerMetrics {
       .labelNames("client", "type")
       .register(registry)
 
-    clientId: ClientId => {
```

Comment:
it is required to do like that, otherwise you will have  errors upon `register` calls from prometheus

Reply:
Reverted.